core/bin/external_node/src/config/mod.rs

```diff
@@ -66,7 +71,12 @@ impl RemoteENConfig {
                 .context("Failed to fetch L1 chain ID")?
                 .as_u64(),
         );
-
+        let base_token_addr = match client.get_base_token_l1_address().await {
+            Err(ClientError::Call(err)) if ErrorCode::MethodNotFound.code() == err.code() => {
+                Address::from_str("0x0000000000000000000000000000000000000001")?
```

Comment:
You can add a comment that it's a fallback for the case when EN version tries to interact with an older main node that doesn't have this method yet.